config/locales/es/activerecord.yml

```diff
@@ -130,8 +130,6 @@ es:
         one: Registro del censo local
         other: Registros del censo local
     attributes:
-      administrator:
-        description: Descripción
```

Comment:
It looks like we shouldn't remove this one but add this key to the English language as well :thinking:.

Reply:
OK, i will check it out

Reply:
Finally removed all changes to Spanish locale files except one that adds a comment so all files in all locales have this comment.